<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditOutFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImageFolder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenImageFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Image Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenOutFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Out Folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMode">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>81</y>
      <width>161</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxShowInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show input</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show output</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScaleBase">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelScaleBase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale base</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScalePower">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelScalePower">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Power</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTiffInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tiff Info</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMatInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mat Info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutocleanOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto clean output text</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLoadAnydepth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Anydepth</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSaveOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Output</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDisplayRange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMinDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>64.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMaxDisp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>16.000000000000000</double>
     </property>
     <property name="value">
      <double>255.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowHist">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Histogram</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxHistScale_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistScaleHeight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistScaleCoef">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>*10^</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelDirHistBarWidth_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar Width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHistBarWidth">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxImRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Rotation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Show mask</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRegexImageFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>.+.tiff</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetImageFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBoxOutputMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowGray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Gray</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowPC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show PC</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxShowGradient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Gradient</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMinDispGrad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>64.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFixMaxDispGrad">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>65535.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>25.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGradientSchape">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="prefix">
      <string>shape </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGradThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>Gradient Th </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxGradient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxProcessGradient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Process Gradient</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxMaskBackGround">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mask Background</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxProcessTile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Process Tile</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTileSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="singleStep">
      <number>32</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileSize">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTileToProcess">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileToProcess">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile To process</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTileCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileCount">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>TileCount</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tile</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelTileScale">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Tile Scale</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxTileScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowTileOnImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Tile On Image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowLesionMask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Show LesionMask</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHorisontalLineMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Horisontal Line Mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
